### PR TITLE
tests: make journalctl check wait a bit for output

### PR DESCRIPTION
**Pull request by mvo5**
The snapd-reexec and catalog-update tests are checking for
journalctl output. However it appears that on some systems like
debian-sid there is a small delay before the output is available.
Give journalctl a bit of extra time in these tests.

### CODE REVIEW

File: tests/main/snapd-reexec/task.yaml [hunk 1]
```diff
@@ -38,6 +38,12 @@ execute: |
     mount --bind /tmp/old-info $SNAP_MOUNT_DIR/core/current/usr/lib/snapd/info
     systemctl start snapd.service snapd.socket
     snap list
+    for i in $(seq 60); do
```
Comment (bboozzoo) [inline tests/main/snapd-reexec/task.yaml]: I think that a comment here and in `tests/main/catalog-update/task.yaml`, documenting why we need the loop would be helpful.

**Comment by sergiocazzolato** [comment]
I like the change but I reproduced the error and after few minutes the test was not in the journalctl output, so, I am  not sure if it is a matter of waiting more time or something else

**Comment by mvo5** [comment]
@sergiocazzolato Oh, thank you. that is good to know

**Comment by bboozzoo** [comment]
Maybe the messages are getting dropped. There's #4411 where rate limiting is supposed to be disabled. OTOH, if they were dropped due to rate limiting there should be a corresponding message indicating that this happened in the journal.

**Comment by mvo5** [comment]
Hm, at least for the snap-reexec it is not slow output, the latest test run waits for 60s but no output appears :(

**Comment by mvo5** [comment]
Closing as even with the wait it will fail sometimes.

**Pull request closed without merging**